### ISSUE TITLE
scrollTime: support to define day in addition to hour

### ISSUE THREAD

**Issue by woodburn8** [fullcalendar/fullcalendar-scheduler#45]
scrollTime currently only supports the hour definition.  When scrollTime is enabled and we are looking at the timelineWeek view, it will scroll to the specified hour of the first day of the week.  This is useless if today is not the first day of the week.  Instead it would be nice if it would scroll to the specified hour of the current day.

**Comment by arshaw** [comment]
i agree. question: would you ever want it to scroll to a certain time that is NOT in the current day? (some other day)

**Comment by woodburn8** [comment]
I'm not sure I would need that myself, but I can see it being useful for some.  For example, those who make Sunday the start of the week on the Calendar might want it scroll to Monday 9am.

What I imagined for scrollTime was the ability to specify an ISO 8601 date/timestamp.  I think that should cover all cases including the one you mentioned.

**Comment by sparh** [comment]
@arshaw  yes I think it could be good to scroll a certain time even if it is not the current day.

For example directly loading a past event. mysite.com/event.php?id=25

**Comment by EliW** [comment]
Similarly, would be good if viewing a 'more than one day' view to be able to scroll to a specific date, instead of reloading everything with a new set of dates.  (In my case, having 3-days of a conference shown.  And wanting to jump to day 2 of it, without it now trying to show 3 different days) :)

**Comment by acerix** [comment]
I found that `scrollTime` works if it is specified in "milliseconds since the beginning of the view range" so that could be calculated based on the date you want to scroll to as a workaround:

https://codepen.io/acerix/pen/KjxXaY?editors=0010
